ophyd/areadetector/filestore_mixins.py

```diff
@@ -169,6 +168,43 @@ def stage(self):
         self._resource = fs.insert_resource('AD_TIFF', self._fp, res_kwargs)
 
 
+class FileStoreTIFFSquashing(FileStoreBase):
+    def __init__(self, *args, images_per_set_name='images_per_set',
+                 number_of_sets_name="number_of_sets",
+                 cam_name='cam1', proc_name='proc1', **kwargs):
```

Comment:
`cam_name`, at least in `ophyd.areadetector.detectors`, is by default `'cam'` (this is the attribute name, not the suffix)